Packs/CoreAlertFields/IncidentFields/incidentfield-Target_Process_Name.json

```diff
@@ -28,5 +28,22 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
-    "x2_fields": "action_process_image_name"
+    "x2_fields": "action_process_image_name",
+    "Aliases": [
+        {
+          "cliName": "childprocess",
+          "type": "multiSelect"
+        },
+        {
+            "cliName": "digitalguardianprocessname",
```

Comment:
I think this refers to process and not child process 
So should be mapped to Initiated By(actor_process_image_name)